common/test/acceptance/tests/lms/test_lms_help.py

```diff
@@ -0,0 +1,118 @@
+"""
+Test Help links in LMS
+"""
+import json
+
+from common.test.acceptance.tests.studio.base_studio_test import ContainerBase
+from common.test.acceptance.pages.lms.instructor_dashboard import InstructorDashboardPage
+from common.test.acceptance.fixtures import LMS_BASE_URL
+
+
+class TestCohortHelp(ContainerBase):
+    """
+    Tests help links in Cohort page
+    """
+    def setUp(self, is_staff=True):
+        super(TestCohortHelp, self).setUp(is_staff=is_staff)
+        self.enable_cohorting(self.course_fixture)
+        self.instructor_dashboard_page = InstructorDashboardPage(self.browser, self.course_id)
+        self.instructor_dashboard_page.visit()
+        self.cohort_management = self.instructor_dashboard_page.select_cohort_management()
+
+    def get_url_with_changed_domain(self, url):
+        """
+        Replaces .org with .io in the url
+        Arguments:
+            url (str): The url to perform replace operation on.
+        Returns:
+        str: The updated url
+        """
+        return url.replace('.org/', '.io/')
+
+    def assert_link(self, expected_link, actual_link):
+        """
+        Assert that 'href' and text inside help DOM element are correct.
+
+        Arguments:
+            expected_link (dict): The expected link attributes.
+            actual_link (dict): The actual link attribute on page.
+        """
+        self.assertEqual(expected_link['href'], actual_link.get_attribute('href'))
+        self.assertEqual(expected_link['text'], actual_link.text)
+
+    def assert_opened_help_link_is_correct(self, url):
```

Comment:
do you have another method that you've already written that does this? I'm wondering if we can make this PR more DRY